mysql-test/suite/innodb/t/percona_changed_page_bmp_flush.test

```diff
@@ -66,18 +66,43 @@ INSERT INTO t1 VALUES (3, REPEAT("c", 20000));
 # Test innodb_flush_method=ALL_O_DIRECT
 # Check that the previous test produced bitmap data while the server is down.
 #
+call mtr.add_suppression("InnoDB: Warning: innodb_log_block_size has been changed from default value");
+call mtr.add_suppression("New log files created");
+
 --exec echo "wait" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
 --shutdown_server 10
 --source include/wait_until_disconnected.inc
+--enable_reconnect
 file_exists $BITMAP_FILE;
 --replace_regex /_[[:digit:]]+\.xdb$//
 list_files $MYSQLD_DATADIR ib_modified_log*;
+--remove_files_wildcard $MYSQLD_DATADIR ib_logfile*
 --enable_reconnect
---exec echo "restart:--innodb-track-changed-pages=1 --innodb-flush-method=ALL_O_DIRECT" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+--exec echo "restart:--innodb-track-changed-pages=0 --innodb-log-block-size=4096 --innodb-flush-method=ALL_O_DIRECT" > $MYSQLTEST_VARDIR/tmp/mysqld.1.expect
+--source include/wait_until_connected_again.inc
+
+RESET CHANGED_PAGE_BITMAPS;
```

Comment:
Maybe just call remove_files_wildcard for bitmaps next to where the log files are removed? This should save a server restart